pcluster/pcluster_stack.py

```diff
@@ -267,7 +274,7 @@ def __init__(self, scope: cdk.Construct, id: str, **kwargs) -> None:
             properties={
                 'Cloud9Environment': cloud9_instance.environment_id,
                 'BootstrapPath': 's3://%s/%s' % (bootstrap_script.s3_bucket_name, bootstrap_script.s3_object_key),
-                'Config': config,
+                'Config': "".join( ['https://', pcluster_config_script.s3_bucket_name,  ".s3.amazonaws.com/", pcluster_config_script.s3_object_key ] ),
```

Comment:
what about if the user specifies a different config file?

Reply:
This only changes the default parameter value to point to the config asset published by upload.sh. It can still be overridden by a user when the stack is launched.

Reply:
My mistake. Reverted to the original config which allows customers to override.